README.md

```diff
@@ -1,19 +1,147 @@
-# Postal on kubernetes
+# postal
 
-A docker-image and helm-chart to deploy [Postal](https://postal.atech.media/) on kubernetes.
+A helm-chart to deploy [Postal](https://github.com/postalserver/postal) on kubernetes.
 
-Please refer to the individual documentation for the docker-image and the helm-chart:
+## Introduction
 
-- [docker-image](docker/README.md)
-- [helm-chart](helm/postal/README.md)
+This chart bootstraps a deployment of Postal, MariaDB and RabbitMQ on a
+[Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-Getting the Docker image and helm chart:
+## Prerequisites
 
-- The docker-image is available on Docker Hub: [linkyard/postal](https://hub.docker.com/r/linkyard/postal/)
-- The helm chart is available at [charts.linkyard.ch](https://charts.linkyard.ch)
+- Kubernetes 1.9
+- PV provisioner support in the underlying infrastructure
 
-## Roadmap
+## Optional prerequisites
 
-- Contribute our little fixes to upstream
-- Add the ability to start individual Portal processes for easier HA setups and scaling
-- Add meaningful readiness and liveness probes
+- An ingress controller
+- A functioning [cert-manager](https://github.com/jetstack/cert-manager) for certificate management
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`, add the helm charts repository:
+
+```console
+clone this repository
+```
+
+Setup an ingress-controller
+
+```console
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+```
+
+```console
+helm install my-ingress ingress-nginx/ingress-nginx `
+     --namespace ingress `
+     --create-namespace `
+     --set controller.replicaCount=2 `
+     --set controller.nodeSelector."beta\.kubernetes\.io/os"=linux `
+     --set defaultBackend.nodeSelector."beta\.kubernetes\.io/os"=linux
+```
+
+Update your domain info using the IP from ingress controller to add to your @/* (A) dns record
+
+Install cert-manager
+
+```console
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.6.1/cert-manager.yaml
+```
+
+Install our included issuer or create your own https://cert-manager.io/docs/concepts/issuer/
+
+```console
+kubectl apply -f issuer-dev.yaml 
+```
+
+When changing from dev to prod
+- Uninstall postal helm chart
+- Delete clusterissuer and it's secret from cert-manager namespace
+- Delete postal-smtp-tls and postal-web-tls from default namespace
+- Install dev or prod issuer then install as normal
+
+Override ingress-controller values to add smtp port
```

Comment:
Does this step need to be after installing cert-manager? If not, I think it would make more sense to group it with the rest of the ingress-controller setup.